ably-rest.gemspec

```diff
@@ -32,25 +32,22 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_runtime_dependency 'faraday', '~> 2.2'
-  spec.add_runtime_dependency 'excon', '~> 0.55'
```

Comment:
is this not needed anymore

Reply:
Nope, seems like ably-ruby stopped using it a long time ago: https://github.com/ably/ably-ruby/commit/0dd0346d99a77bd25b78c0be1baa6ea9aa15d4ea